Common/SonarQube.Plugins.Common.csproj

```diff
@@ -61,7 +61,7 @@
     <Compile Include="Utilities.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="SonarQube\License.txt" />
```

Comment:
A couple of the projects had license files under a "SonarQube" folder. This is left-over from the original, pre-joint project prototype: I copied bits of code from the Scanner, and I put any code I'd copied in a "SonarQube" folder and included the license.
We should now clean up and re-organise some of the code and folders e.g. get rid of these "SonarQube" sub-folders, and move some of the non-Roslyn-specific classes out of the Roslyn-specific project. However, that can all way until after v1.0.